powertrack-rb.gemspec

```diff
@@ -0,0 +1,27 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'powertrack/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'powertrack-rb'
```

Comment:
You should call the gem 'powertrack' as long it's a rubygems. But keep the repository named powertrack-rb.